lib/ansible/modules/cloud/vultr/vultr_network.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2018, Yanis Guenane <yanis+ansible@guenane.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: vultr_network
+short_description: Manages networks on Vultr.
+description:
+  - Manage networks on Vultr. Note: A network cannot be updated. It needs to be deleted and re-created.
+version_added: "2.7"
+author: "Yanis Guenane (@Spredzy)"
+options:
+  name:
+    description:
+      - Name of the network.
+    required: true
+    aliases: [ description, label ]
+  cidr:
+    description:
+      - The CIDR IPv4 network block to be used when attaching servers to this network
+    required: true
+  region:
+    description:
+      - Region the network is deployed into.
+    required: true
```

Comment:
please remove `required: true`and replace with a description:

~~~
- Required if I(state=present).
~~~